<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFRenderToImagesDialog</class>
 <widget class="QDialog" name="PDFRenderToImagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render Document to Images</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <widget class="QGroupBox" name="filesGroupBox">
     <property name="title">
      <string>Output Files Settings</string>
     </property>
     <layout class="QGridLayout" name="filesGroupBoxLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pagesToRenderLabel">
        <property name="text">
         <string>Pages to render</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="fileTemplateLabel">
        <property name="text">
         <string>File template</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="pagesAllButton">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="directoryLabel">
        <property name="text">
         <string>Generate into directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="selectDirectoryButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="directoryEdit"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="fileTemplateEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="pagesSelectButton">
        <property name="text">
         <string>Pages:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="selectedPagesEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="imageResolutionGroupBox">
     <property name="title">
      <string>Image Resolution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="resolutionLabel">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="resolutionPixelsButton">
        <property name="text">
         <string>Pixels</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="resolutionDPIButton">
        <property name="text">
         <string>DPI</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="resolutionDPIEdit">
        <property name="suffix">
         <string> dots/inch</string>
        </property>
        <property name="minimum">
         <number>72</number>
        </property>
        <property name="maximum">
         <number>6000</number>
        </property>
        <property name="singleStep">
         <number>36</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="resolutionPixelsEdit">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>16384</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="imageFormatSettings">
     <property name="title">
      <string>Image Format Settings</string>
     </property>
     <layout class="QGridLayout" name="imageFormatSettingsLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="imageFormatLabel">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="imageSubtypeLabel">
        <property name="text">
         <string>Subtype</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="subtypeComboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="qualityEdit">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="formatComboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="gammaLabel">
        <property name="text">
         <string>Gamma</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="optimizedWriteLabel">
        <property name="text">
         <string>Optimized write</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="gammaEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="qualityLabel">
        <property name="text">
         <string>Quality</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="compressionLabel">
        <property name="text">
         <string>Compression</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="compressionEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="progressiveScanWriteLabel">
        <property name="text">
         <string>Progressive scan write</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="optimizedWriteCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="progressiveScanWriteCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PDFRenderToImagesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PDFRenderToImagesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
